Source/WebCore/rendering/PositionedLayoutConstraints.cpp

```diff
@@ -397,23 +396,60 @@ void PositionedLayoutConstraints::computeBlockStaticDistance(const RenderBox& re
         m_insetBefore.setValue(LengthType::Fixed, staticLogicalTop);
 }
 
+void PositionedLayoutConstraints::fixupLogicalLeftPosition(RenderBox::LogicalExtentComputedValues& computedValues) const
```

Comment:
`adjust` is the WebKit term here.

Reply:
... I feel like "adjust" is too generic. You can adjust for a lot of reasons. Adjust for relpos. Adjust for alignment. Adjust to switch from content-box sizing to border-box sizing... What are we adjusting for? Do we need to? Every time I see a function like this I am wondering.

"Fixup" is like, fix this thing because it's broken as-is. Which feels a lot closer to what this function is doing.

Anyway that's my pitch for keeping it as-is. :) Lmk if I was sufficiently convincing. :D